scraper/src/github-scraper/parseEvents.ts

```diff
@@ -105,19 +118,22 @@ async function addCollaborations(event: PullRequestEvent, eventTime: Date) {
 
   if (collaborators.size > 1) {
     const collaboratorArray = Array.from(collaborators); // Convert Set to Array
-    for (const user of collaboratorArray) {
-      const others = new Set(collaborators);
-      const othersArray = Array.from(others);
-
-      others.delete(user);
-      appendEvent(user, {
-        type: "pr_collaborated",
-        title: `${event.repo.name}#${event.payload.pull_request.number}`,
-        time: eventTime.toISOString(),
-        link: event.payload.pull_request.html_url,
-        text: event.payload.pull_request.title,
-        collaborated_with: [...othersArray],
-      });
+    if (event.payload.pull_request.base.ref === default_branch) {
+      for (const user of collaboratorArray) {
+        const others = new Set(collaborators);
+        const othersArray = Array.from(others);
+
+        others.delete(user);
+
+        appendEvent(user, {
+          type: "pr_collaborated",
+          title: `${event.repo.name}#${event.payload.pull_request.number}`,
+          time: eventTime.toISOString(),
+          link: event.payload.pull_request.html_url,
+          text: event.payload.pull_request.title,
+          collaborated_with: [...othersArray],
+        });
+      }
```

Comment:
the check can be merged along with line before like `if (collaborators.size > 1 && event.payload... === default branch` right?